File: interp.mjs
import {argv} from 'process';
import {readFileSync} from 'fs';
import {Env} from './temper.out/ebb/js/interp.js';

const env = new Env();

env.source(String(readFileSync("ebb.ebb")));
env.call("main-low", [String(readFileSync(argv[2]))]);

File: interp.py
import logging
import sys

logging.basicConfig(filename="/dev/stdout", level=logging.DEBUG, format="%(message)s")

from ebb.interp import Env

def read(name):
    with open(name) as f:
        return f.read()

env = Env()

env.source(read("ebb.ebb"))
env.call("main-low", (read(sys.argv[1]),))
